indico/modules/events/timetable/client/js/types.ts

```diff
@@ -51,6 +76,7 @@ export interface BreakEntry extends BaseEntry, ScheduledMixin {
   sessionId?: number;
   textColor: string;
   backgroundColor: string;
+  personLinks?: PersonLink[];
```

Comment:
Maybe making this non-optional but defaulting to an empty array is better? Is there any case when `personLinks` are not defined at all?